app/controllers/admin/trips_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Admin::TripsController < Admin::BaseController
+  def edit
+    @trip = Trip.find(params[:id])
+  end
+
+  def update
+    trip = Trip.find(params[:id])
+    if trip.update(trip_params)
+      flash[:success] = "Trip #{trip.id} was updated"
```

Comment:
I would like to put together a list of css for flash messages. Any ideas?

Reply:
That can be easily added as the css classes are already built-in (.notice, .success, etc).  I will open a chore in pivotal for that.